src/assets/scripts/InputController.js

```diff
@@ -447,7 +447,7 @@ export default class InputController {
 
             case KEY_CODES.LEFT_ARROW:
                 // shortKeys in use
-                if (prop.game.option.get('controlMethod') === 'arrows') {
+                if (window.gameController.game.option.get('controlMethod') === 'arrows') {
                     this.$commandInput.val(`${currentCommandInputValue} \u2BA2`);
```

Comment:
👍 for moving these to the `gameController`!

Side note: Out of scope for you changes, but, there looks to be _a lot_ of room for enumerations in the `InputController`.  I'll add an issue for that (#123)